contracts/sophia.md

```diff
@@ -898,6 +898,7 @@ Data is encoded in memory as follows:
     - `option(t)`: `{4, encode(t)}`
     - `datatype(cs)`: `{5, encode(cs)}`, where `cs : list(list(typerep))`
       are the type representations for the constructor arguments
+  - The `ttl` type is encoded as the datatype `datatype ttl = RelativeTTL(int) | FixedTTL(int)`.
```

Comment:
I see that this line, joined with the description above

> Values of datatypes are encoded as tuples where the first component is a constructor tag (starting with 0 for the first constructor), and the following components are the constructor arguments.

address https://github.com/aeternity/epoch/pull/1493#discussion_r216269522

Thanks.